src/script/page/AppLock/AppLock.tsx

```diff
@@ -156,11 +159,16 @@ const AppLock: React.FC<AppLockProps> = ({
     app?.style.setProperty('pointer-events', isVisible ? 'none' : 'auto', 'important');
 
     if (isVisible) {
-      modalObserver.observe(document.querySelector('#wire-main'), {
-        childList: true,
-        subtree: true,
```

Comment:
I was unable to pinpoint the exact cause, but this observer was checking for both `childList` and `subtree` for a removal attempt of the modal.
It doesn't seem to be necessary to include the subtree with our current code structure, as the app correctly logs out when removing the app lock modal from the DOM